prowler/providers/azure/services/network/network_http_internet_access_restricted/network_http_internet_access_restricted.py

```diff
@@ -0,0 +1,29 @@
+from prowler.lib.check.models import Check, Check_Report_Azure
+from prowler.providers.azure.services.network.network_client import network_client
+
+
+class network_http_internet_access_restricted(Check):
+    def execute(self) -> Check_Report_Azure:
+        findings = []
+        for subscription, security_groups in network_client.security_groups.items():
+            for security_group in security_groups:
+                report = Check_Report_Azure(self.metadata())
+                report.subscription = subscription
+                report.resource_name = security_group.name
+                report.resource_id = security_group.id
+                report.status = "PASS"
+                report.status_extended = f"Security Group {security_group.name} from subscription {subscription} has HTTP internet access restricted."
+                rule_fail_condition = any(
+                    rule.destination_port_range == "80"
```

Comment:
Please, review if the rule accepts all ports or a range of ports that includes port 80.

Reply:
Done